src/_base.scss

```diff
@@ -0,0 +1,58 @@
+/**
+  * Base Styles
+  * To use in SCSS with the @extend & @include command.
```

Comment:
Can we document this in readme too, just so it's super clear how you'd use this?